appcenter-data/android/src/main/java/com/microsoft/appcenter/reactnative/data/AppCenterReactNativeDataModule.java

```diff
@@ -68,22 +68,45 @@ public void read(String documentId, String partition, ReadableMap readOptionsMap
         } else {
             readOptions = new ReadOptions(TimeToLive.DEFAULT);
         }
-        Data.read(documentId, JsonElement.class, partition, readOptions).thenAccept(new Consumer("Read failed", promise));
+        Data.read(documentId, JsonElement.class, partition, readOptions).thenAccept(new Consumer<JsonElement>("Read failed", promise));
     }
 
     @ReactMethod
-    public void create(String documentId, ReadableMap readableMap, String partition, ReadableMap writeOptionsMap, final Promise promise) {
+    public void create(String documentId, ReadableMap documentMap, String partition, ReadableMap writeOptionsMap, final Promise promise) {
```

Comment:
Let's align the order of parameters with `read` and `remove` APIs.
```suggestion
    public void create(String documentId, String partition, ReadableMap documentMap, ReadableMap writeOptionsMap, final Promise promise) {
```

Reply:
Done